packages/slice-machine/.npmignore

```diff
@@ -26,3 +26,7 @@ tests/project/node_modules
 
 # prov
 /api-dump
+
+# source maps
+/build/**/*.map
```

Comment:
hey, i'm not sure but maybe source map have been added for Sentry ?
Do you know if Sentry still works if we don't have the source maps ? Or is it maybe that source maps are uploaded when creating a Sentry release ?

Reply:
Ok, 👌 

I don't know much about how source map works so one last question.
If an error is triggered in Sentry, will it still be able to link the error with the source map ?

Reply:
Thanks